cocos/audio/win32/AudioEngine-win32.cpp

```diff
@@ -83,12 +83,11 @@ bool AudioEngineImpl::init()
         s_ALDevice = alcOpenDevice(NULL);
         
         if (s_ALDevice) {
-            auto alError = alGetError();
```

Comment:
@dumganhar may be we need to check the error after invoking `alcCreateContext ` or `alcMakeContextCurrent `?